src/Microsoft.Build.Utilities.ProjectCreation/Package.cs

```diff
@@ -462,18 +462,18 @@ internal void WriteNuspec(Stream stream)
             writer.WriteElementStringIfNotNull("copyright", Copyright);
             writer.WriteElementStringIfNotNull("tags", Tags);
             writer.WriteElementStringIfNotNull("projectUrl", ProjectUrl);
-            writer.WriteElementStringIfNotNull("requireLicenseAcceptance", RequireLicenseAcceptance.ToString());
+            writer.WriteElementStringIfNotNull("requireLicenseAcceptance", RequireLicenseAcceptance.ToString().ToLowerInvariant());
             writer.WriteElementStringIfNotNull("licenseUrl", LicenseUrl);
             writer.WriteElementStringIfNotNull("icon", Icon);
             writer.WriteElementStringIfNotNull("iconUrl", IconUrl);
-            writer.WriteElementStringIfNotNull("developmentDependency", DevelopmentDependency.ToString());
+            writer.WriteElementStringIfNotNull("developmentDependency", DevelopmentDependency.ToString().ToLowerInvariant());
```

Comment:
What do you think about adding an overload `WriteElementStringIfNotNull(string, bool)` which sets the value to `true/false` in lower case?  I'm just worried there's some value here that we missed and making it more obvious might be better.  Thoughts?

```c#
internal static void WriteElementStringIfNotNull(this XmlWriter writer, string localName, bool value)
{
    writer.WriteElementString(localName, value ? "true" : "false");
}
```

Reply:
Yeah, I like that idea. Will update now.

Reply:
I prefer PRs to be split up by individual fixes or implementations just so its easier to review the impact.  This PR is a great example of fixing a problem and adding a unit test, which is my preference.

Reply:
Updated!